EduPerfTests/Program.cs

```diff
@@ -158,13 +230,17 @@ static void DeterminePerformanceIterations()
             if (performanceIterations == 0) DeterminePerformanceIterations();
         }
 
-        static void DeterminePageLoadSites()
+        static void DetermineSitesToLoad()
         {
-            Console.WriteLine("Specify the path to the csv file containing the sites you would like to test (blank to skip):");
-            var path = Console.ReadLine();
+            Console.WriteLine("Specify the path to the csv file containing the sites you would like to test (blank to skip, d for default):");
+            pathToSites = Console.ReadLine();
+
+            if (string.IsNullOrWhiteSpace(pathToSites)) return;
 
-            if (string.IsNullOrWhiteSpace(path)) return;
-            using (var reader = new StreamReader(path))
+            if (pathToSites.ToLower() == "d")
+                pathToSites = @"c:\users\johnjan\desktop\sitelist.csv";
```

Comment:
This should probably be generalized. Either something off the root drive (although this isn't foolproof) or more favorably relative to the current directory (so it works on anyone's machines). Then we can inform the user where they can place a file to get some defaults (and have a sample.cvs for a default placed there).